emc2/simulator/radar_moments.py

```diff
@@ -263,6 +264,7 @@ def calc_radar_moments(instrument, model, is_conv,
                 sigma_d_numer = [x for x in map(_calc_sigma_d_liq, np.arange(0, Dims[1], 1))]
 
             sigma_d_numer_tot = np.nan_to_num(np.stack([x[0] for x in sigma_d_numer], axis=1))
+            print(np.all(sigma_d_numer_tot == 0))
```

Comment:
I think you're trying to debug the sigma_d issue, but I wound up tracking it down. So I don't think we need these print statements.

Reply:
I agree. good catch!